package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "bookstore",
```

Comment:
- [ ] _**I am using this place to make comments.**_  ---> This is great that you want to merge your commits into `develop `from `feature/setup` branch. However, to use `Gitflow` you have to make the `develop ` a default branch. Kindly fix it 🙏